composer.patches.json

```
@@ -27,6 +27,9 @@
     "drupal/environment_indicator": {
       "d.o. #3456599": "https://www.drupal.org/files/issues/2024-06-24/3456599-6.patch"
     },
+    "drupal/webp": {
+      "d.o. #3281606": "https://git.drupalcode.org/project/webp/-/merge_requests/33.patch"
```

```suggestion
      "d.o. #3281606": "https://git.drupalcode.org/project/webp/-/merge_requests/33.patch"
```

It would be better to save the patch locally I think in case git.drupalcode.org is unavailable.

Also I think it would be better to use the .diff file: https://git.drupalcode.org/project/webp/-/merge_requests/33.diff instead of a .patch. But you can save it locally as a *.patch file.

---

I think we can still download modules from drupal.org but git.drupalcode.org is more of a source code hosting. IIRC sometimes it gets stuck asking for a key because of rate limits n stuff. But I might be misremembering.